keps/168-2-pending-workloads-visibility/README.md

```diff
@@ -257,17 +259,20 @@ We plan on adding sanity e2e tests, and RBAC e2e tests. The e2e RBAC tests shoul
 
 ### Graduation Criteria
 
-First iteration (0.6):
+#### Alpha
 - Release the new API in alpha. This allows us to adjust the API according to users' and reviewers' feedback,
 - Release it with a feature gate.
 
-Second iteration (0.7):
-- Release the API in beta and guarantee backwards compatibility,
-- Reconsider introducing a throttling mechanism based on user and review feedback,
-- Consider introducing FlowScheme and PriorityLevelConfiguration to allow admins to easily tune API priorities.
+#### Beta
```

Comment:
Please also add merging the `visibility-api.yaml` into the main yaml. 

One potential complication might be about disabling the feature. We should check experimentally if the [issue](https://github.com/kubernetes-sigs/kueue/issues/1519) when we disabled the feature gate. If this is the case we may need to document the procedure of disabling the feature.

cc @PBundyra

Reply:
> Please also add merging the visibility-api.yaml into the main yaml.

Yes, this is a good point. Thanks!

Reply:
> One potential complication might be about disabling the feature. We should check experimentally if the https://github.com/kubernetes-sigs/kueue/issues/1519 when we disabled the feature gate. If this is the case we may need to document the procedure of disabling the feature.

I tried to run it with and without VisibilityOnDemand. Looks like works fine.

Is I correct understood flow? Run with --feature-gates=VisibilityOnDemand=false and try to get `kubectl get resourceflavor`. Is it correct?

Reply:
> Is I correct understood flow? Run with --feature-gates=VisibilityOnDemand=false and try to get kubectl get resourceflavor. Is it correct?

Yes, this is what I wanted to test. Can you also check if the visibility endpoint gets disabled then? And if you can re-enable the feature, and it works as expected. 

If all the tests succeed I think we are good to merge-in the visibility-api.yaml.

Reply:
awesome

Reply:
In the past, there were some errors printed everytime you used kubectl. But I think that was fixed.

I guess it's possible that there are some error logs in Kueue, but hopefully users don't get too confused with them.

Reply:
> In the past, there were some errors printed everytime you used kubectl. But I think that was fixed.
> 
> I guess it's possible that there are some error logs in Kueue, but hopefully users don't get too confused with them.

I think yes, because I can't see some errors on logs.